src/workerd/io/worker.c++

```diff
@@ -2026,9 +2026,9 @@ void Worker::handleLog(jsg::Lock& js,
     auto formatLog = formatLogVal.As<v8::Function>();
 
     auto levelStr = logLevelToString(level);
-    args[length] = v8::Boolean::New(js.v8Isolate, colors);
-    args[length + 1] = v8::Boolean::New(js.v8Isolate, bool(loggingOptions.structuredLogging));
-    args[length + 2] = jsg::v8StrIntern(js.v8Isolate, levelStr);
+    args[length] = js.boolean(colors);
+    args[length + 1] = js.boolean(bool(loggingOptions.structuredLogging));
```

Comment:
Hmm what's with the dodgy function-style cast? What is the type of loggingOptions.structuredLogging?

Reply:
We can also just have a variant of `js.boolean()` that takes a structured bool.